CONTRIBUTING.md

```diff
@@ -20,21 +20,30 @@ This subfolder contains the raw html and css files that are used to set the fram
 
 E.g. if a qiskit project (with the `qiskit_sphinx_theme` installed) wants to override any of the themes files (such as the `layout.html`) that the `qiskit_sphinx_theme` provides it needs to have its own version of that file (e.g. `layout.html`) stored in its `docs/_templates` folder.
 
-### `/docs`
-This subfolder contains a scaled down version of the Sphinx build that builds the documentation for the Qiskit repos. It pulls styles from the `/qiskit_sphinx_theme` subfolder. You can check any changes you are making to the `qiskit_sphinx_theme` by building the documentation (see running locally section) and opening the html files generated in `/docs/_build/html`.
+### `/example_docs`
+This subfolder contains a scaled down version of the Sphinx build that builds the documentation for the Qiskit repos. 
 
-### `/docs/sphinx_guide`
-This subfolder contains some example `.rst` files that show how to implement specific Sphinx features such as class documentation, panel layouts, images etc.
+It pulls styles from the `/qiskit_sphinx_theme` subfolder. You can check any changes you are making to the `qiskit_sphinx_theme` by building the documentation (see running locally section) and opening the html files generated in `/example_docs/docs/_build/html`.
+
+### `/docs_guide`
+This subfolder contains instructions on how to use Sphinx in Qiskit projects.
 
 ------
 ## Run locally
 
-1. Set up a Python environment to work in, e.g.:
-   1. `python3 -m venv .venv/`
-   2. `source .venv/bin/activate`
-2. Run `pip install -r requirements-dev.txt`
-3. Run `pip install -e .`
-4. Run `sphinx-build -b html example_docs/docs/ example_docs/docs/_build/html` to build the documentation. This will create the `example_docs/docs/_build` subfolder where you can see the html files generated based on the `.rst` files in the `docs` folder and the styles pulled from the `qiskit_sphinx_theme`.
+We use [Tox](https://tox.wiki/en/latest/), which you will need to install globally (e.g. using [`pipx`](https://pypa.github.io/pipx/)).
+
+* Build `example_docs/`:
+  1. `tox -e py`
+  2. Open up `example_docs/docs/_build/html/index.html` in your browser
+* Build `docs_guide`:
+  1. `tox -e docs-guide`
+  2. Open up `docs_guide/_build/html/index.html` in your browser.
+* Run doctests for the docs guide: `tox -e doctest`
+
+Sometimes Sphinx's caching can get in a bad state. First, try running `tox -e docs-clean`, which will remove Sphinx's cache. If you are still having issues, try adding `-r` your command, e.g. `tox -e py -r`. `-r` tells Tox to reinstall the dependencies.
+
+To build the Furo theme, use `THEME=_qiskit_furo` in front of the command, e.g. `THEME=_qiskit_furo tox -e py`.
```

Comment:
I think we need a teeny bit more explanation about furo here. E.g. explain that there are 2 themes being actively worked on and currently the non-furo styles are pulled in by default when you run the build commands